SurveyApp/Modules/Splash/SplashViewController.swift

```diff
@@ -83,8 +84,15 @@ extension SplashViewController {
     }
 
     private func beginAnimation() {
-        UIView.animate(withDuration: 1.0, delay: 0.5) { [weak self] in
-            self?.logoImageView.alpha = 1.0
-        }
+        UIView.animate(
+            withDuration: 1.0,
+            delay: 0.5,
+            animations: { [weak self] in
+                self?.logoImageView.alpha = 1.0
+            },
+            completion: { [weak output] _ in
+                output?.showNextScreen()
```

Comment:
The name isn't right IMO, it should describe an output from the `beginAnimation` action
But with `showNextScreen` it likes you decide the logic here, that when `beginAnimation` it has to show the next screen
```suggestion
                output?.animationDidFinish()
```